src/auto/functions.rs

```diff
@@ -198,21 +193,15 @@ pub fn print_run_page_setup_dialog<P: IsA<Window>>(
     }
 }
 
-pub fn print_run_page_setup_dialog_async<
-    P: IsA<Window>,
-    Q: FnOnce(&PageSetup) + Send + Sync + 'static,
```

Comment:
Hrm, this looks wrong

Reply:
See https://github.com/gtk-rs/gir/pull/981

Reply:
If you update gir another time, it should be all good now :)

Reply:
Done